scandium-core/src/test/java/org/eclipse/californium/scandium/DTLSConnectorTest.java

```diff
@@ -1468,22 +1477,31 @@ public void testConnectorAbortsHandshakeOnUnknownPskIdentity() throws Exception
 			.setPskStore(new StaticPskStore("unknownIdentity", CLIENT_IDENTITY_SECRET.getBytes()))
 			.build();
 		client = new DTLSConnector(clientConfig);
-		client.setErrorHandler(new ErrorHandler() {
+		client.start();
+		final AtomicReference<AlertMessage> alert = new AtomicReference<>();
+		MessageCallback callback = new MessageCallback() {
 
 			@Override
-			public void onError(InetSocketAddress peerAddress, AlertLevel level, AlertDescription description) {
-				latch.countDown();
-				assertThat(level, is(AlertLevel.FATAL));
-				assertThat(peerAddress, is(serverEndpoint));
+			public void onSent() {
 			}
-		});
-		client.start();
-		SimpleMessageCallback callback = new SimpleMessageCallback();
+
+			@Override
+			public void onError(Throwable error) {
+				if (HandshakeException.class.isInstance(error)) {
```

Comment:
Is `error instanceof HandshakeException` out of style?

Reply:
No, not out of style :-) I just got used to the style we are using in Hono. But I will happily change it to the style you seem to have gotten accustomed to. However, I would have actually expected some sympathy from you based on your ongoing usage of e.g.
```
if (0 == variable) {
...
```
right? ;-)

Reply:
I have updated the PR accordingly ...